03-setup-ingress-controller/nginx.md

```diff
@@ -388,22 +388,22 @@ DESCRIPTION:
   documentation.
 ```
 
-You can define `multiple rules` for different `hosts` and `paths` in a single `ingress` resource. To keep things organized (and for better visibility), `Starter Kit` tutorial provides two ingress manifests for each host: [echo](assets/manifests/echo_host_nginx.yaml) and [quote](assets/manifests/quote_host_nginx.yaml).
+You can define `multiple rules` for different `hosts` and `paths` in a single `ingress` resource. To keep things organized (and for better visibility), `Starter Kit` tutorial provides two ingress manifests for each host: [echo](assets/manifests/echo_host.yaml) and [quote](assets/manifests/quote_host.yaml).
```

Comment:
Manifest file path for the `echo` and `quote` hosts is incorrect. Should be:

```
[echo](assets/manifests/nginx/echo_host.yaml)
[quote](assets/manifests/nginx/quote_host.yaml)
```

Reply:
Applied the fix